File: code/src/test/scala/cern/colt/matrix/tlong/impl/SparseRCLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseRCLongMatrix2DViewTest(arg0: String) extends SparseRCLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseRCLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseRCLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseRCLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseRCLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class SparseRCLongMatrix2DViewTest extends SparseRCLongMatrix2DTest {

    public SparseRCLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
